api/users.go

```diff
@@ -36,10 +39,11 @@ func GetUsers(c *gin.Context) {
 func GetUser(c *gin.Context) {
 	user, err := store.GetUserLogin(c, c.Param("login"))
 	if err != nil {
-		c.String(404, "Cannot find user. %s", err)
-	} else {
-		c.JSON(200, user)
+		c.AbortWithError(http.StatusNotFound, fmt.Errorf("Cannot find user. %s", err))
```

Comment:
@tboerger That would be better. I propose something like:
```json
{
  "errors": [{
    "status": "400",
    "detail": "JSON parse error - Expecting property name at line 1 column 2 (char 1)."
  }]
}
```
An array of errors makes sense for validation.